docs/source/conf.py

```diff
@@ -22,7 +22,7 @@
 author = "Rose K. Cersonsky, Guillaume Fraux, Sergei Kliavinek, Alexander Goscinski, Benjamin Helfrect, and Michele Ceriotti"
 
 # The full version, including alpha/beta/rc tags
-release = "0.0.1"
+release = "0.1.0"
```

Comment:
I meant literally importing skcosmo in this script, and setting `release = skcosmo.__version__`, so that we don't have to update it here when releasing a new version

Reply:
Done :D